docs/scripts/link-checker-puppeteer.ts

```diff
@@ -51,7 +51,10 @@ function reportResult(links: LinkInfo[]) {
         }
       }
     );
-    throw new Error(`${errorLinks.length} broken links found`);
+    const failedLinks = errorLinks.filter((link) => link.statusCode >= 400);
+    if (failedLinks.length) {
+      throw new Error(`${errorLinks.length} broken links found`);
```

Comment:
Nit: Should this be failedLinks.length?

Reply:
Done 2d1d1956afbeabd1ae3f3cf4f4872521c1da839e